ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/operations/VirtualOperation.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.hyperledger.besu.ethereum.vm.operations;
+
+import org.hyperledger.besu.ethereum.core.Gas;
+import org.hyperledger.besu.ethereum.vm.MessageFrame;
+import org.hyperledger.besu.ethereum.vm.Operation;
+
+public class VirtualOperation implements Operation {
```

Comment:
The virtual operations changes should be in a separate standalone PR.  While tracing uses it ultimately it is an orthogonal change.

Reply:
should i revert the commit then ?

Reply:
Work it in another pr and then bring it back in with a merge.

Reply:
ok sounds good.